idaes/core/surrogate/pysmo/tests/test_kriging.py

```diff
@@ -97,6 +97,9 @@ def test__init__07(self, array_type):
 
     @pytest.mark.unit
     @pytest.mark.parametrize("array_type", [np.array, pd.DataFrame])
+    @pytest.mark.filterwarnings(
+        "ignore:invalid value encountered in log:RuntimeWarning"
```

Comment:
Just for my understanding, where are these warnings coming from and are we sure that they do not indicate an underlying issue somewhere?

Reply:
The warnings are coming from zeros and NaNs when computing logs. The specific warning is coming from this computation:

```
 RuntimeWarning: invalid value encountered in log:
 log_like = (0.5 * ns * np.log(ssd)) + (0.5 * lndetcov)
```
The warnings are expected: ssd and lndetcov involve matrix manipulations (e.g. multiplication, diagonalization, cholesky factorization), and in some cases that may lead to NaNs or zeros during computation.